.github/workflows/broken-links-checker.yml

```diff
@@ -0,0 +1,19 @@
+name: Broken links checker
+on:
+  push:
+    paths:
+      - '.github/workflows/broken-links-checker.yml'
+  schedule:
+    - cron: '*/28 6 * * *' # Run once daily at arbitrary time (6:28 AM UTC)
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '12'
+      - run: npm install broken-link-checker -g
+      - name: Run broken-link-checker on neptune.ai
+        run: blc https://neptune.ai -ro --requests 5 --host-requests 5 --exclude 'https://*.linkedin.com/**' --get
+      - name: Run broken-link-checker on docs.neptune.ai
```

Comment:
It seems that this step does not run when the first link check (for neptune.ai) outputs errors.
If we want to keep them both here (and I still think it makes sense) we can either change the order or ensure that both of those steps run independently.